main/reference/zoe-api/zoe.md

```diff
@@ -3,300 +3,333 @@
+The **adminFacet** has one method:
+- **getVatShutdownPromise()**    
+  - Returns a promise that resolves to reason (the value passed to **fail(reason)**) or 
+    completion (the value passed to **exit(completion)**) when this newly started instance terminates. 
+
+A **publicFacet** is an object available via Zoe to anyone knowing
+the instance they are associated with. The **publicFacet** is used for general queries
 and actions, such as getting a current price or creating public invitations. Since a
 facet is defined just as any other object,
 the contract developer can add methods to them just like they would any object.
 
-The `creatorFacet` is only available in this return value (i.e. only when starting
+The **creatorFacet** is only available in this return value (i.e. only when starting
 a contract instance). The contract designer
 should use it to encapsulate things that the contract runner might not want to share,
 or might want to control the distribution of. The party who starts the contract
-should carefully consider the impact before sharing access to the `creatorFacet`.
+should carefully consider the impact before sharing access to the **creatorFacet**.
 
-`creatorInvitation` is an invitation that the contract instance creator can use.
+**creatorInvitation** is an invitation that the contract instance creator can use.
 It is usually used in contracts where the creator immediately sells
 something (auctions, swaps, etc.), so it's helpful for the creator to have
 an invitation to escrow and sell goods. Remember that Zoe invitations are
-represented as a `Payment`.
+represented as a **Payment**.
 ```js
 const issuerKeywordRecord = {
-  'Asset': moolaIssuer,
-  'Price': quatlooIssuer
+  Asset: moolaIssuer,
+  Price: quatlooIssuer
 };
 const terms = { numBids: 3 };
-const { creatorFacet, publicFacet, creatorInvitation } = await E(zoe).startInstance(
+const { creatorFacet, publicFacet, creatorInvitation } = await E(Zoe).startInstance(
   installation, issuerKeywordRecord, terms);
 ```
-## `E(Zoe).offer(invitation, proposal, paymentKeywordRecord, offerArgs)`
-- `invitation` `{Invitation|Promise<Invitation>}`
-- `proposal` `{Proposal}` - optional
-- `paymentKeywordRecord` `{PaymentKeywordRecord}` - optional
-- `offerArgs` `{Object}` - optional
-- Returns: `{Promise<UserSeat>}`
-
-Used to make an offer to the contract that created the `invitation` that is
+
+## E(Zoe).offer(invitation, proposal?, paymentKeywordRecord?, offerArgs)
+- **invitation** **Invitation|Promise&lt;Invitation>**
+- **proposal** **Proposal** - Optional.
+- **paymentKeywordRecord** **PaymentKeywordRecord** - Optional.
+- **offerArgs** **Object**
+- Returns: **Promise&lt;[UserSeat](./user-seat.md)>**
+
+Used to make an offer to the contract that created the **invitation** that is
 provided as the first argument.
 
 ::: tip Vat per Contract
 All contracts each run in a new vat with their own version of the
```

Comment:
Is this the best place for this tip? Seems more like maybe this should be included in the `startInstance` docs. 
Also, why use a `tip` at all? Could this not simply be part of the documentation for the correct method?

Reply:
Agreed on both points. I moved that text into the first paragraph of the StartInstance method.